ember/app/routes/flight/change-pilot.js

```diff
@@ -12,7 +12,7 @@ export default Ember.Route.extend({
     let accountId = this.get('account.user.id');
     let clubId = this.get('account.club.id');
     let clubMembers = clubId ? this.get('ajax').request(`/users?club=${clubId}`)
-      .then(it => it.users.filter(user => user.id !== accountId)) : [];
+      .then(it => it.users.filter(user => user.id !== accountId).filter(user => user.upload_permission_club)) : [];
```

Comment:
First i thought so, but `user.upload_permission_club` is correct.
Compare skylines/frontend/views/users.py